Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -106,9 +106,8 @@ RefPtr<GStreamerPeerConnectionBackend> GStreamerMediaEndpoint::peerConnectionBac
 GStreamerMediaEndpoint::NetSimOptions GStreamerMediaEndpoint::netSimOptionsFromEnvironment(ASCIILiteral optionsEnvVarName)
 {
     NetSimOptions options;
-    auto tokens = StringView::fromLatin1(g_getenv(optionsEnvVarName));
-    for (auto it : tokens.split(',')) {
-        auto option = it.toString();
+    auto tokens = String::fromUTF8(g_getenv(optionsEnvVarName));
+    for (auto option : tokens.split(',')) {
```

Comment:
Is there some reason for this change? The original avoids making a copy of the environment variable contents into a `String`. This is not a big deal because it does not seem to be performance-critical code; but I am curious.